src/AspNetCoreModuleV1/AspNetCore/aspnetcoremodule.rc

```diff
@@ -3,6 +3,7 @@
 #include <windows.h>
 #include "version.h"
 #include "resource.h"
+#include "Aspnetcore_msg.rc"
```

Comment:
Nit: The actual filename starts with lowercase 'a'.  (Not a huge deal in Windows, but let's better be correct.)